BaragonAgentService/src/main/java/com/hubspot/baragon/agent/managers/AgentRequestManager.java

```diff
@@ -136,7 +136,7 @@ public Response processRequest(String requestId, Optional<RequestAction> maybeAc
 
   private Response reload(BaragonRequest request, boolean delayReload) throws Exception {
     if (!delayReload) {
-      configHelper.checkAndReload();
+      configHelper.reloadConfigs();
```

Comment:
i think this one we still want to check. This is part of the flow for individual updates coming from baragon service, not the flow of bootstrapping. Maybe want to split out reloadConfigs and checkAndReload so we can call those independently